lib/application.js

```diff
@@ -27,6 +27,7 @@ var resolve = require('path').resolve;
 var Router = require('router');
 var setPrototypeOf = require('setprototypeof')
 var slice = Array.prototype.slice;
+var router = require("router");
```

Comment:
There is already an import for `router` above; you can just use that same import.

Reply:
Man I must be blind 🤦‍♂️